pkg/resolver/messages.go

```diff
@@ -0,0 +1,83 @@
+package resolver
+
+import (
+	"context"
+
+	"github.com/doug-martin/goqu/v9"
+	"github.com/doug-martin/goqu/v9/exp"
+	"github.com/stolostron/search-v2-api/graph/model"
+	db "github.com/stolostron/search-v2-api/pkg/database"
+	klog "k8s.io/klog/v2"
+)
+
+func Messages(ctx context.Context) ([]*model.Message, error) {
+	searchSchemaResult := &SearchSchemaMessage{
+		pool: db.GetConnection(),
+	}
+	searchSchemaResult.messageQuery(ctx)
+	return searchSchemaResult.messageResults(ctx)
+}
+
+func (s *SearchSchemaMessage) messageQuery(ctx context.Context) {
```

Comment:
- We only have 1 message for now, but I want to prepare to support other messages.
- Match the naming of equivalent function on search.go

```suggestion
// Build the query to find any ManagedClusters where the search addon is disabled.
func (s *SearchSchemaMessage) buildSearchAddonDisabledQuery(ctx context.Context) {
```